library/src/main/java/com/chuckerteam/chucker/internal/data/entity/HttpTransaction.kt

```diff
@@ -191,12 +191,16 @@ internal class HttpTransaction(
     }
 
     private fun formatBody(body: String, contentType: String?): String {
-        return when {
-            contentType != null && contentType.contains("json", ignoreCase = true) ->
-                FormatUtils.formatJson(body)
-            contentType != null && contentType.contains("xml", ignoreCase = true) ->
-                FormatUtils.formatXml(body)
-            else -> body
+        return if (contentType.isNullOrBlank()) {
+            body
+        } else {
+            when {
+                contentType.contains("json", ignoreCase = true) -> FormatUtils.formatJson(body)
+                contentType.contains("xml", ignoreCase = true) -> FormatUtils.formatXml(body)
+                contentType.contains("x-www-form-urlencoded", ignoreCase = true) ->
+                    FormatUtils.formatUrlEncodedForm(body)
+                else -> body
+            }
```

Comment:
```suggestion
        return when {
            contentType.isNullOrBlank() -> body
            contentType.contains("json", ignoreCase = true) -> FormatUtils.formatJson(body)
            contentType.contains("xml", ignoreCase = true) -> FormatUtils.formatXml(body)
            contentType.contains("x-www-form-urlencoded", ignoreCase = true) ->
                FormatUtils.formatUrlEncodedForm(body)
            else -> body
        }
```

Please keep the single `return when` style as it's easier to read and follow.